src/BUILD

```diff
@@ -522,6 +522,7 @@ cc_library(
             "libovms_threadsafequeue",
             "//:ovms_dependencies", # TODO dispose or replace
             "//src/kfserving_api:kfserving_api_cpp",
+            "ovms_capimodule",
```

Comment:
why do we change name convention? libcapimodule

Reply:
this would produce liblibcapimodule. THere is undergoing refactor and after that I would remove all those prefixes (libovms/ovms) since its obvious that it is from ovms (lack of @ prefix).